pom.xml

```diff
@@ -93,6 +93,7 @@
 
   <modules>
     <module>common</module>
+    <module>format</module>
```

Comment:
I have change the format to run all the test i am adding this otherwise build will fail, after passing all the testcases i will remove this